bluetti_mqtt/mqtt_client.py

```diff
@@ -210,7 +210,18 @@ def payload(id: str, device: BluettiDevice, **kwargs) -> str:
                             .encode(),
                         retain=True
                         )
-
+            await client.publish(f'homeassistant/select/{d.sn}_led_mode/config',
+                        payload=payload(
+                            id='led_mode',
+                            device=d,
+                            name='LED Mode',
+                            icon='mdi:lightbulb',
+                            value_template= r'{{ value_json.power_on_behavior }}',
```

Comment:
Neither am I. The result is besrt explained with the screenshot below. It seems like we can tell homeassistant that we have a "select" field with the listed options. Additionally i set the icon to look more like a light-control. I chose this approach over the various light integrations in home assistant since that would require separating On/Off from the effect (LOW, HIGH, SOS). 

![image](https://user-images.githubusercontent.com/6293002/192568728-a480d853-bb20-40ad-9414-2d9bf582106c.png)

Reply:
Reading through the documentation on `value_template` I'm fairly certain that it's not necessary. My understanding is that if you're posting JSON to that MQTT topic, the `value_template` can allow you to extract the specific field you want. In their example for a sensor they say:

> Setting up a sensor with multiple measurement values requires multiple consecutive configuration topic submissions.
> * Configuration topic no1: `homeassistant/sensor/sensorBedroomT/config`
> * Configuration payload no1: `{"device_class": "temperature", "name": "Temperature", "state_topic": "homeassistant/sensor/sensorBedroom/state", "unit_of_measurement": "°C", "value_template": "{{ value_json.temperature}}" }`
> * Configuration topic no2: `homeassistant/sensor/sensorBedroomH/config`
> * Configuration payload no2: `{"device_class": "humidity", "name": "Humidity", "state_topic": "homeassistant/sensor/sensorBedroom/state", "unit_of_measurement": "%", "value_template": "{{ value_json.humidity}}" }`
> * Common state payload: `{ "temperature": 23.20, "humidity": 43.70 }`

Because this switch is just a simple string value, it should just work without the need for a `value_template`.

Reply:
You are right. I tried it without and it still works :)